content/docs/reference/pkg/nodejs/pulumi/aws/_index.md

```diff
@@ -304,38 +304,10 @@ deployments.
 Use this data source to get the ID of a registered AMI for use in other
 resources.
 
-## Example Usage
```

Comment:
It looks like examples are missing in all these re-generated docs. I'm guessing the `tf2pulumi` tool isn't found on your `$PATH`?

On my various machines, I've cloned https://github.com/pulumi/tf2pulumi in my go directory, and then I run `make ensure` and `make build && make install`.